src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -741,6 +962,19 @@ class MultiSign_test : public beast::unit_test::suite
                     "fails local checks: Invalid signature.");
         }
         {
+            // Single-sign, but invalidate the transaction.
```

Comment:
It looks like this is a valid single sign test case, but I'm wondering if it belongs in the MultiSign file?  However I don't feel strongly about it.

I wonder if the comment should read, "Single-sign, but invalidate the sequence number."?